### PR TITLE
Major bump of live-common

### PR DESCRIPTION
**Pull request by gre**
this is to test https://github.com/LedgerHQ/ledger-live-common/pull/189

It contains many preparation work for Q2, especially for the incoming libcore ethereum transition, erc20 and send max implementations.

### Type

preparation

### Context

literally Q2

### Parts of the app affected / Test plan

- **Basically everything should work like before apart from the point below**
- There are some performance improvement
- There is a new Experimental Features: Scan for invalid paths. It scan for 2 new cases on Bitcoin derivations: segwit on legacy and legacy on segwit. This is an opt-in because we don't want to automatically scan for these for all users. That means path that are invalid are no longer scanned, however, path that are considered "legacy" are still scanned regardless. The logic behind is that if Ledger ever allowed to import such account it's legacy, otherwise it's invalid because user's fault.
- There is a slight change on the naming that you will see, you will see account names to contain `(segwit)` where it was not the case before. This is to prepare what we'll have later displayed as a "visual tag" instead of being part of the name.

### CODE REVIEW

**Comment by gre** [comment]
> Invalid accounts should be retrieved during "Add account" process:

as explained in the description, you need to enable a experimental feature, have you? @Arnaud97234

**Comment by gre** [comment]
@Arnaud97234  fixed

**Comment by gre** [comment]
ready to review again

File: src/components/modals/Debug.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ class Debug extends Component<*, *> {
       for (let x = 0; x < 20; x++) {
         const { address, path } = await getAddress
           .send({
+            derivationMode: 'segwit',
```
Comment (juan-cortes) [inline src/components/modals/Debug.js]: `account.derivationMode` ?

Comment (gre) [reply src/components/modals/Debug.js]: in context of this Debug screen, it seems we only do segwit derivations. https://github.com/LedgerHQ/ledger-live-desktop/blob/7027e6316d5734166c1b8863c4a656791f631830/src/components/modals/Debug.js#L49

**Pull request merged**